idd/Energy+.idd.in

```diff
@@ -92182,7 +92182,7 @@ ElectricLoadCenter:Storage:Battery,
         \minimum 5
         \default 10
         \note Only required when battery life calculation is activated
-   A7; \field Battery Life Curve Name
+   A7;  \field Battery Life Curve Name
```

Comment:
OK, so while it technically is an IDDChange, it's not a functional IDD change.  Got it.